getting_started/data_backup.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Data Backup
+layout: default
+parent: Getting Started
+nav_order: 3
+---
+
+# Data Backup
+
+While data storage on Arjuna is [RAID] configured to provide robustness against disk failure, disaster can strike (and did, RIP a lot of things 2021), so it's individual users' responsibility to **back up your data regularly**, _especially_ things that would take a lot of (human and/or computer) time to reproduce! There are a few approaches you can take to do this (outlined on this page), and you will likely want to choose a combination of them for various types of files/code/data. **We strongly recommend that users configure backups to run in a scheduled, automatic fashion** (e.g. using cron jobs) to avoid the need to remember to back up and the possibility of attendant data loss. Instructions to set this up are provided below.
```

Comment:
Maybe drop "and did, RIP a lot of things 2021"

Reply:
haha I leave it to you

Reply:
I think it adds a sense of urgency which might help encourage people to do it. Everyone always says anything can happen, but knowing that it has happened here makes it that much more likely to happen

Reply:
Actually I am going to request a small change. Leave the reference to the incident in, but get rid of the RIP 2021 part. Just say "and has in the past" or something

Reply:
`(and [has](https://lists.andrew.cmu.edu/mailman/private/arjuna-users/2021-September/000050.html))`

Reply:
Yeah! that's perfect